internal/monitors/appmesh/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+monitors:
+- dimensions: null
+  doc: |
+
+    This monitor starts a StatsD monitor to listen to StatsD metrics emitted
+    by AWS AppMesh Envoy Proxy.
+
+    Sample YAML configuration:
+
+    ```yaml
+    monitors:
+     - type: appmesh
+       listenAddress: 0.0.0.0
+       listenPort: 8125
+       metricPrefix: statsd.appmesh
+    ```
+
+  metrics: null
```

Comment:
Fill out the expected metric set so they are documented for users.

Reply:
Done.